### PR TITLE
MINOR: Clean-up streams javadoc warnings

### PR DESCRIPTION
**Pull request by bbejeck**
*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
ok to test

**Comment by bbejeck** [comment]
ping one of @abbccdda, @ableegoldman, @mjsax, @vvcephei  for review

**Review by vvcephei — APPROVED** [review]
Thanks for the fixes!

File: streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java [hunk 1]
```diff
@@ -184,9 +184,9 @@
      * <pre>
      * KafkaStreams streams = ... // some aggregation on value type double
      * String queryableStoreName = "storeName" // the store name should be the name of the store as defined by the Materialized instance
-     * ReadOnlyKeyValueStore<K, ValueAndTimestamp<VOut>> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<K, ValueAndTimestamp<VOut>> timestampedKeyValueStore());
+     * ReadOnlyKeyValueStore&gt;K, ValueAndTimestamp&gt;VOut&lt;&lt; localStore = streams.store(queryableStoreName, QueryableStoreTypes.&gt;K, ValueAndTimestamp&gt;VOut&lt;&lt; timestampedKeyValueStore());
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java]: I seem the right fix would be to add the missing `{@code }` annotation in L184?
```
<pre>{@code
```
The closing `}` is already in L190.

File: streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java [hunk 2]
```diff
@@ -236,9 +236,9 @@
      * <pre>
      * KafkaStreams streams = ... // some aggregation on value type double
      * String queryableStoreName = "storeName" // the store name should be the name of the store as defined by the Materialized instance
-     * ReadOnlyKeyValueStore<K, ValueAndTimestamp<VOut>> localStore = streams.store(queryableStoreName, QueryableStoreTypes.<K, ValueAndTimestamp<VOut>> timestampedKeyValueStore());
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java:239]: As above.

**Review by mjsax — APPROVED** [review]
On suggestion. Feel free to merge. And thanks for the cleanup!

File: streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java (hunk 1 above)
Comment (bbejeck) [reply streams/src/main/java/org/apache/kafka/streams/kstream/CogroupedKStream.java]: Ah, good catch I'll update this and the next.

**Comment by bbejeck** [comment]
java doc changes only, so merging this

**Comment by bbejeck** [comment]
merged #9461 into trunk

**Comment by bbejeck** [comment]
cherry-picked to 2.7

**Pull request merged**